content/docs/command-reference/dag.md

```diff
@@ -1,49 +1,50 @@
 # dag
 
-Show [stages](/doc/command-reference/run) in a pipeline that lead to the
-specified stage. By default it lists
-[DVC-files](/doc/user-guide/dvc-files-and-directories).
+Visualize the pipeline(s) in
+[`dvc.yaml`](/doc/user-guide/dvc-files-and-directories#dvclock-file) as a graph
+of connected [stages](/doc/command-reference/run).
 
 ## Synopsis
 
 ```usage
 usage: dvc dag [-h] [-q | -v] [--dot] [--full] [target]
 
 positional arguments:
-  targets         Stage or output to show pipeline for (optional)
-                  Finds all stages in the workspace by default.
+  target          Stage or output to show pipeline for (optional)
+                  Uses all stages in the workspace by default.
 ```
 
 ## Description
 
 A data pipeline, in general, is a series of data processing
 [stages](/doc/command-reference/run) (for example console commands that take an
 input and produce an <abbr>output</abbr>). A pipeline may produce intermediate
-data, and has a final result. Machine learning (ML) pipelines typically start a
-with large raw datasets, include intermediate featurization and training stages,
-and produce a final model, as well as accuracy
-[metrics](/doc/command-reference/metrics).
+data, and has a final result.
+
+Data processing or ML pipelines typically start a with large raw datasets,
```

Comment:
all this text is pretty bad :( let's create a ticket to rewrite it since it's def not a focus of this PR.

Reply:
Extracted to #1500